src/api/app/models/kiwi/package_group.rb

```diff
@@ -0,0 +1,27 @@
+class Kiwi::PackageGroup < ApplicationRecord
+  has_many :packages
+  belongs_to :image
+
+  # we need to add a prefix, to avoid generating class methods that already
+  # exist in Active Record, such as "delete"
+  enum kind: %i[bootstrap delete docker image iso lxc oem pxe split testsuite vmx], _prefix: :kind
```

Comment:
But if its called `type` in the kiwi description why not just keep that naming and call it `type` in the model too?